css-media-queries/meta-viewport.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <!-- your code here 👇 -->
+  <meta name="viewport" content="initial-scale=1, maximum-scale=1">
```

Comment:
The `content` attribute here is not correct 🤔 

It's value should be `"width=device-width, initial-scale=1.0"`